coder_sniffer/Drupal/Sniffs/Commenting/InlineVariableCommentSniff.php

```diff
@@ -83,16 +83,18 @@ public function process(File $phpcsFile, $stackPtr)
             if (strpos($tokens[$stackPtr]['content'], '@var') !== false) {
                 $warning = 'Inline @var declarations should use the /** */ delimiters';
 
-                if ($phpcsFile->addFixableWarning($warning, $stackPtr, 'VarInline') === true) {
-                    if (strpos($tokens[$stackPtr]['content'], '#') === 0) {
-                        $varComment = substr_replace(rtrim($tokens[$stackPtr]['content']), '/**', 0, 1)." */\n";
-                    } else if (strpos($tokens[$stackPtr]['content'], '//') === 0) {
-                        $varComment = substr_replace(rtrim($tokens[$stackPtr]['content']), '/**', 0, 2)." */\n";
-                    } else {
-                        $varComment = substr_replace($tokens[$stackPtr]['content'], '/**', 0, 2);
+                if (strpos($tokens[$stackPtr]['content'], '#') === 0 || strpos($tokens[$stackPtr]['content'], '//') === 0) {
```

Comment:
I find this harder to read than the original. Maybe add a short comment? Also creating variable `$varComment` was clear before, but now you just do the replaceToken() directly with nested rtrim/ltrim. So a source comment explaining it would be very helpful

Reply:
Fair point, I've re-added the `$varComment` variables the to replace the one-liner and added comments why the trims are needed.